include/hnswlib/hnswalg.h

```diff
@@ -1191,6 +1191,34 @@ namespace hnswlib {
             return cur_c;
         };
 
+        std::priority_queue<std::pair<dist_t, labeltype >>
+        searchSubset(const void *query_data, const void *subset_data, size_t k, size_t n) const {
```

Comment:
who is the caller of this?

Reply:
Why u need to pass pointer and not a reference to the data?

Reply:
`searchSubset` is an internal function. There will be an entry function of searching(e.g., `search(...)`). In this function, a simple strategy of searching method will be implemented:
1. less data (eg.., 1024) to search from -> use `searchSubset()` 
2. use hnsw search method

Here, no special reason to use a pointer. Just want to keep the code style consistent with original hnswlib.

Reply:
Then it is much safer to use `references` better than `pointers`